crates/api_models/src/payments.rs

```diff
@@ -1179,7 +1179,7 @@ pub struct CtpServiceDetails {
     pub provider: Option<api_enums::CtpServiceProvider>,
     /// Encrypted payload
     #[schema(value_type = Option<String>)]
-    pub encypted_payload: Option<Secret<String>>,
+    pub encrypted_payload: Option<Secret<String>>,
```

Comment:
This could technically be an API contract change, depending on how / where it's being used (request or response).

Reply:
did a global search and update. i believe it has been updated everywhere. yes, this will indeed be an api contract change.